### ISSUE TITLE
Can’t see my current issues

### ISSUE THREAD

**Issue by espressobeanies** [CodeHubApp/CodeHub#545]
Can’t see acct issues via iOS 12.1.2 app

**Comment by Chewsterchew** [comment]
Good point, no commits for 5 months. This used to be so active, what happened?

**Comment by garrettw** [comment]
I bet he just got too busy with life. I know how it goes.

**Comment by espressobeanies** [comment]
I wish open-source projects would be more inclined to setup a donation page to keep things going

**Comment by espressobeanies** [comment]
Thanks Dillon!

**Comment by thedillonb** [comment]
@espressobeanies, I can't to reproduce this on latest iOS - I still see all my issues. By any chance, are you looking for issues that belong to an organization? If so, did you happen to grant that organization third-party access? If you don't do that then issues for any repository within that organization do not appear.

**Comment by espressobeanies** [comment]
No, these are my personal repos. I can check my permission settings for the
CodeHub app to verify.
